jest-transform.cjs

```diff
@@ -0,0 +1,18 @@
+const path = require("path");
+
+/**
+ * Jest cannot process images imported within the code. This loads their path as modules for test purpose.
+ *
+ * e.g.
+ * import loaderUrl from "../../assets/loader.webp";
+ */
+module.exports = {
+  process(sourceText, sourcePath, options) {
+    return {
+      code: `module.exports = ${JSON.stringify(path.basename(sourcePath))};`,
+    };
+  },
```

Comment:
should this somehow be limited by file extension?

Reply:
It's done by the transformer pattern of the configuration

```
"\\.(svg|png|webp)$": "<rootDir>/jest-transform.cjs",
```